src/FactionsPro/FactionCommands.php

```diff
@@ -91,7 +100,7 @@ public function onCommand(CommandSender $sender, Command $command, $label, array
                                         $task = new FactionWar($this->plugin, $r);
                                         $handler = $this->plugin->getServer()->getScheduler()->scheduleDelayedTask($task, 20 * 60 * 2);
                                         $task->setHandler($handler);
-                                        $p->sendMessage("The war against $factionName and $sFaction has started!");
+                                        $p->sendMessage("The war against " . $factionName . " and " . $sFaction . " has started!");
```

Comment:
If you want to do it like that in your own plugins it's fine, of course; but a PR to  another repo that just changes things to how  you prefer them is not likely to be accepted by anyone, unless you can give a good reason such as performance or readability.

This commit and the one removing the 'use' statements add nothing, and fix nothing.